<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>ファイル</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>編集</string>
    </property>
    <addaction name="actionMerge"/>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionOpacity"/>
    <addaction name="actionUseResult"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>開く</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="text">
    <string>合成</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionOpacity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>半透明化</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>戻す</string>
   </property>
  </action>
  <action name="actionUseResult">
   <property name="text">
    <string>合成結果を使用する</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
